components/incoming/AboutIcxIntern.vue

```diff
@@ -4,8 +4,7 @@
     <div class="about-main">
       <div class="about-content">
         <IconInAssetsPath
-          :data="icons[0]"
-          :key="icons.src"
+          :data="icon_support"
```

Comment:
判別しやすいように名称を設定（命名規則的なのもあった方が良いかな..）

Reply:
[IMO]
もっとシンプルに変数定義すら必要ない気もしている。

これはイメージなので取り込まないで欲しいけどこんな感じ


```suggestion
          :data="{src: "images/icx/tech-support.svg", alt: ""}"
```